Added functional nav bars

julia-999 opened the pull request:
- not linked to the right pages yet
- icons change size a little bit, I'm playing around with the svg files themselves to see if I can make them all the same

julia-999 committed:
Added functional nav bars

- not linked to the right pages yet
- icons change size a little bit, I'm playing around with the svg files themselves to see if I can make them all the same

jhwan7 commented on src/Containers/Pages/MainRecipeModal/MainRecipeModal.js:
```diff
@@ -50,6 +53,11 @@ class MainRecipeModal extends React.Component {
     return (
       <div className={styles.fullPage}>
+
+{/* Top nav bar */}
```
you dont need comments here

jhwan7 commented on src/Containers/Pages/MainRecipeModal/MainRecipeModal.js:
```diff
@@ -85,6 +93,11 @@ class MainRecipeModal extends React.Component {
           onClick={() => addLikedRecipe(this.state.spoonacularRecipe.id)}
         ></input>
+
+{/* bottom nav bar */}
```
same here

jhwan7 commented on src/component/navbars/BottomNavbar.js:
```diff
@@ -0,0 +1,39 @@
+import heartFilled from './heart_filled.svg';
+import homeFilled from './home_filled.svg';
+import personFilled from './person_filled.svg';
+import bookmarkFilled from './bookmark_filled.svg';
```
these should all be stored inside /asset/image folder

jhwan7 commented on src/component/navbars/Navbar.module.scss:
```diff
@@ -0,0 +1,39 @@
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    // overflow: hidden;
```
remove comment

jhwan7 commented on src/component/navbars/TopNavbar.js:
```diff
@@ -0,0 +1,19 @@
+    <div className={styles.topNavContainer}>
+        <NavItem image={arrow} route='back'/>
+        <NavItem image={platepal} route={routes.login}/>
+        <NavItem image={menu} route={routes.metrics}/>
```
is metrics the right path?

jhwan7 commented on src/Containers/Pages/ProfilePage/ProfilePage.js:
```diff
@@ -6,6 +6,8 @@ import ProfileRestrictionInfo from '../../../component/ProfileRestrictionInfo/Pr
 import LogoutBtn from '../../../component/LogoutBtn/LogoutBtn.js';
 import ProfileMetricInfo from '../../../component/ProfileMetricInfo/ProfileMetricInfo.js';
+import TopNavbar from '../../../component/navbars/TopNavbar';
+import BottomNavbar from '../../../component/navbars/BottomNavbar';
```
these two things are repetitive, so instead of having these 2 elements on every page, its better to have a wrapping component. the "fill" attribute can be managed by the wrapping component

julia-999 committed:
Updated navbar implementation

- combined top and bottom navbar
- removed comments
- made it rendered in app.js

julia-999 committed:
Merge branch 'main' into feature/navbars
